### ISSUE TITLE
Mortar Stone Windows (weird effects) 

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#8875]
**Version:** 0.7.6.3 beta

in building underwater, if i place Mortar Stone Windows (or any other half width block) i get weird graphic effects if i stand right next to it. ( i think the following are all to do with Air still being part of the block)

Normal View underwater
![snap2348](https://user-images.githubusercontent.com/29383891/43795007-f171df3e-9a77-11e8-9b5c-1e183f65f6d5.png)
backed up against the Window 
![snap2349](https://user-images.githubusercontent.com/29383891/43795020-fa4ba3a6-9a77-11e8-9258-ea533147e533.png)

Also if i have like a wall of mortar Stone windows and swim right up to any block higher that one block i will **FALL** underwater to the floor.
![snap2350](https://user-images.githubusercontent.com/29383891/43795289-d18f4db8-9a78-11e8-96dd-85e0af578c31.png)

if i place windows without a corner piece i can physically go through this gap
these two snapshots are underwater and then inside
![snap2353](https://user-images.githubusercontent.com/29383891/43795415-30866da6-9a79-11e8-8802-da59c0cd7b43.png)
![snap2354](https://user-images.githubusercontent.com/29383891/43795417-309f6888-9a79-11e8-9099-9349398cf2b6.png)

This does **NOT** happen with a Half block wall
![snap2356](https://user-images.githubusercontent.com/29383891/43795858-7f5903de-9a7a-11e8-86f1-11167e42aca6.png)

**Comment by keegan-o** [comment]
None of these problems are things I'd change by changing the mortared stone blocks-- that you can get through a corner gap with the windows is fine in my book-- but the water just doesn't know how to deal with any unique shape of builders.

The strange effects by the windows are because you are half in and half out of the wall of water, not anything changing the mortared stone stuff would fix-- the only thing we could do to potentially help a bit is change the collider to a full box, but that's not the philosophy with Eco blocks so I wouldn't want to do that just for this (would have to change countless others, and then players would complain of being blocked by invisible geometry all the time). 

So this gets filed under water itself being improved someday to be more reactive, better defined, less glitchy/comprised differently than it currently is.

**Comment by redwyre** [comment]
Water only knows if a block is there or not, I don't think it would even run up to a wall or anything less than full width.

**Comment by Conchipedia** [comment]
i agree with @keegan-o  we can live with this effect until water can be looked at in the Future as not many get this problem as not many people build in or under water
